fixtures/custom-src-paths/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@fixtures/custom-src-paths",
+  "private": true,
+  "main": "./main.js",
```

Comment:
Can this one be removed too? Think we are only requiring the sku config by path

Reply:
Ah, missed this one.